celery/backends/base.py

```diff
@@ -647,11 +647,36 @@ def _forget(self, task_id):
 
     def _store_result(self, task_id, result, state,
                       traceback=None, request=None, **kwargs):
+
+        if state in self.READY_STATES:
+            date_done = datetime.datetime.utcnow()
```

Comment:
Probably rename to `completed_at` ?

Reply:
Done.

Reply:
Not important, I just noted this because it encodes the type (`date`) when it actually is `datetime` and I find `done` too generic, even `ready` could be better (as in `READY_STATES`) :smile:

Reply:
@georgepsarakis I'd like to make all the backends consistent if possible -- should we leave this as date_done, or change it (date_ready?) in the database backend as well (maybe make a backwards-compatible reference in database backend?)

Reply:
You are very right. I found references of `date_done` in MongoDB, Cassandra and RDBMS backends. I think you can leave it as is then, sorry!